person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name: 'Unknown', parents_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parents_permission = parents_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private
+
+  def of_age
+    @age >= 18
+  end
+
+  def can_use_services?
+    @parent_permission || of_age?
+  end
```

Comment:
Nice job implementing the `of_age` method as a private method👍🏼. But kindly note that the `can_use_service` is meant to be a public method😉. I suggest you take the `can_use_service` method to go above the `private` declaration.

```suggestion

  def can_use_services?
    @parent_permission || of_age?
  end
  
  private

  def of_age
    @age >= 18
  end

```

Reply:
okay @Ibizugbe, I'll do that now, thank you